components/NavigationBar/src/MDCNavigationBar.m

```diff
@@ -449,22 +449,22 @@ - (CGRect)mdc_frameAlignedVertically:(CGRect)frame
                            alignment:(UIControlContentVerticalAlignment)alignment {
   switch (alignment) {
     case UIControlContentVerticalAlignmentBottom:
-      return CGRectMake(frame.origin.x, CGRectGetMaxY(bounds) - frame.size.height, frame.size.width,
-                        frame.size.height);
+      return CGRectMake(frame.origin.x, CGRectGetMaxY(bounds) - CGRectGetHeight(frame), CGRectGetWidth(frame),
```

Comment:
This line is over 100 columns, and needs to be split up.  My suggestion would be to place each argument on a separate line.

```objectivec
return CGRectMake(frame.origin.x,
                  CGRectGetMaxY(bounds) - CGRectGetHeight(frame),
                  CGRectGetWidth(frame),
                  CGRectGetHeight(frame));
```